cmd/gopogh/main.go

```diff
@@ -89,3 +98,10 @@ func main() {
 		fmt.Println(string(j))
 	}
 }
+
+// dbVarProvided checks whether any of the database flags/environment variables are set
+func dbVarProvided(dbPath string, dbBackend string) bool {
+	return (dbPath != "" || os.Getenv("DB_PATH") != "") ||
+		(dbBackend != "" || os.Getenv("DB_BACKEND") != "")
```

Comment:
Not that it really matters, but since you just care if any of those are populated they don't need to be contained in `()`

Reply:
Yeah I know. Medya said to add the parentheses for readability